Scripts/coin.gd

```diff
@@ -1,8 +1,8 @@
 extends Area2D
 
 # Set range/speed of coinbob
-var bob_height : float = 5.0
-var bob_speed : float = 5.0
+const _bob_height : float = 5.0
+const _bob_speed : float = 5.0
```

Comment:
so for naming conventions, we want single underline snakecase, variable type declaration? noted

Reply:
an underline is for private variables, since Godot has no concept of public and private variables and functions it's recommended to do it this way

Reply:
oh, what about globally accessible/static variables ?
so we use _foo_bar for private, and foo_bar for public ?

Reply:
yeah we simply use the dash at the start when we don't want a variable to be accessed outside its class, even for global objects they may have their own public and private variables and functions, for Autoloads which are the user defined global objects in Godot we can access them by their name, for example `XManager.varialbe`